### ISSUE TITLE
bluetooth config in readme doesn't work

### ISSUE THREAD

**Issue by jvaleski** [philscott-dev/homebridge-stagg-ekg-plus-server#4]
unfortunately the "sudo setcap" command is not permitted using the default homebridge setup/terminal.

**Comment by philscott-dev** [comment]
This server is meant to live on separate hardware from your homebridge installation. I recommend a Pi Zero W in the readme.

**Comment by jvaleski** [comment]
feel like I'm getting close :). pi zero w's are hard to come by... I have one on order, but it won't be here for a week. I'm trying to setup on a pi 3 b+, and am running into various issues. I'm installing Pi OS 32-bit (Debian Bullseye) from the raspberry pi imager v1.7.1. from there, I have to `apt-get install npm`, and then it feels like a handful of other dependencies fail. a notable example seems like `npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"arm"})` which I get when trying to install pm2

I'm ultimately able to get node running and the server response to /api/status requests, BUT, it's clearly not connecting w/ the kettle (who's mac address I find via bluetoothctl scan) as power state is always '0' even when I manually turn it on.

any guidance appreciated.

**Comment by philscott-dev** [comment]
You may want to try with Pi OS (Buster). I don't _think_ it should matter, but I did this work on Buster and I haven't tried Bullseye. I also did not go through apt to get npm. The ARM binary you install should come with npm. I wonder if something weird has happened here? Hard to say. I've never used [this guide](https://www.makersupplies.sg/blogs/tutorials/how-to-install-node-js-and-npm-on-the-raspberry-pi) to setup node on a pi, but at quick glance it looks like the right steps to follow.

Another thing to try (and you may already be doing this), is to do your troubleshooting with all of your devices in close proximity of each other. I realized way too late that I was having 2 different proximity and signal strength problems, with bluetooth and with wifi, while developing.

Report back with any progress though! I'll do what I can to help.

**Comment by jvaleski** [comment]
Thank you! looking forward to trying Buster… I’ll be a few days until I can report back though; thanks!

**Comment by jvaleski** [comment]
had a chance to try this before heading out; it worked!

- imaged the Pi OS Buster Desktop Legacy version (Lite didn't work).
- followed the node/npm setup/install guide you posted above to get those binaries installed
- followed the readme instructions and those went through without a hitch (warnings and some minor version stuff, but, I ignored those)


however, it took a reboot to get the server to connect to the kettle *and* pm2 doesn't seem to be setup properly as I was assuming it would handle server startup automatically on reboot, but, I had to manually start it back up. after that though, the /api/status HTTP call is showing all the right stuff.

I'll full-loop test this (homekit, homebridge, plugin, server) this weekend.

thanks!
